### DISCUSSION TITLE
How to handle multiple line breakes?

### DISCUSSION THREAD

**Discussion by JoaoGuedesRev** [mdx-editor/editor#618] [Q&A]
Whenever I use .setMarkdown method, if the markdown I pass contains multiple line breaks (\n\n for example), those are condensed into one. How can I change this behavior? I only see a toMarkdownOptions but not a fromMarkdownOptions. Did someone else stumbled in the same issue?

**Comment by pkelaita** [comment]
I have this same issue! Did you ever figure it out?

**Comment by mahdi-openup** [comment]
same here

**Comment by JoaoGuedesRev** [comment]
I solved this issue by transforming the string before passing it to setMarkdown. I did something like this:

`
   markdownString.replace(/\n{2,}/g, (match) => {
        return '<br/>'.repeat(match.length - 1) + '\n\n';
    });
`

And when applying the changes, transforming it back using:

`markdownString.replace(/<br\s*\/?>/g, '\n');`

**Comment by petyosi** [comment]
https://github.com/mdx-editor/editor/issues/112
